### PR TITLE
Exporter 1.0.0b25

### PR DESCRIPTION
**Pull request by jeremydvoss**
# Description

Release exporter with sampler fix to enabled attach sampling

# All SDK Contribution checklist:
- [x] **The pull request does not introduce [breaking changes]**
- [x] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md).**

## General Guidelines and Best Practices
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md##building-and-testing)
- [x] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/monitor/azure-monitor-opentelemetry-exporter/setup.py [hunk 1]
```diff
@@ -85,8 +85,8 @@
         "azure-core<2.0.0,>=1.28.0",
         "fixedint==0.1.6",
         "msrest>=0.6.10",
-        "opentelemetry-api~=1.21",
-        "opentelemetry-sdk~=1.21",
+        "opentelemetry-api~=1.24",
```
Comment (lzchen) [inline sdk/monitor/azure-monitor-opentelemetry-exporter/setup.py:88]: Any reason for the opentelemetry bumps?

Comment (jeremydvoss) [reply sdk/monitor/azure-monitor-opentelemetry-exporter/setup.py:88]: Not specifically. Don't we always want to keep them updated? If not, I can wait unti I make the upstream autoinstrumentation changes.

Comment (lzchen) [reply sdk/monitor/azure-monitor-opentelemetry-exporter/setup.py:88]: This is a min version. There is no reason to keep them "updated" if there is no change in upstream that was released that needs to be supported explicitly.

Comment (jeremydvoss) [reply sdk/monitor/azure-monitor-opentelemetry-exporter/setup.py]: changed back

File: sdk/monitor/azure-monitor-opentelemetry-exporter/CHANGELOG.md [hunk 2]
```diff
@@ -1,14 +1,11 @@
 # Release History
 
-## 1.0.0b25 (Unreleased)
+## 1.0.0b25 (2024-05-07)
```
Comment (lzchen) [inline sdk/monitor/azure-monitor-opentelemetry-exporter/CHANGELOG.md:3]: Wrong date.

**Review by lzchen — APPROVED** [review]
Please fix date

**Pull request merged**